hypothesis-python/src/hypothesis/extra/array_api.py

```diff
@@ -269,6 +287,30 @@ def check_valid_minmax(prefix, val, info_obj):
             kw["exclude_max"] = exclude_max
 
         return st.floats(width=finfo.bits, **kw)
+    else:
+        try:
+            float32 = xp.float32
+            float64 = xp.float64
+            complex64 = xp.complex64
+            complex128 = xp.complex64
```

Comment:
Will explore a "generic" way of doing this, i.e. so you could technically have `complex64` work if you only had `xp.float32` and not `xp.float64`. Generally this could look nicer.

Reply:
Relevant https://github.com/numpy/numpy/issues/22260

Reply:
Eh, thought to just keep with the lazy solution and wrote an appropriate comment and error message.